IaC/AKS/README.md

```diff
@@ -157,7 +162,10 @@ az feature register --name AIWorkspacePreview --namespace microsoft.insights
 az provider register -n microsoft.insights
 
 # Create App Insights
-az monitor app-insights component create -g $Ngsa_App_RG -l $Ngsa_Location -a $Ngsa_Name -o table
+
```

Comment:
since we're using log analytics, I believe this can be removed. See the note in triage for discussion